service/internal/auth/casbin.go

```diff
@@ -87,10 +87,13 @@ p,	role:standard,		/entityresolution/resolve,							write,  	allow
 # Public routes
 ## gRPC routes
 ## for ERS, right now we don't care about requester role, just that a valid jwt is provided when the OPA engine calls (enforced in the ERS itself, not casbin)
-p,	role:unknown,			entityresolution.EntityResolutionService.ResolveEntities,										write,		allow
+p,	role:unknown,			entityresolution.EntityResolutionService.ResolveEntities,					write,		allow
+p,	role:unknown,     kas.AccessService/Rewrap, 			                                  write,	  allow
 ## HTTP routes
 ## for ERS, right now we don't care about requester role, just that a valid jwt is provided when the OPA engine calls (enforced in the ERS itself, not casbin)
-p,	role:unknown,			/entityresolution/resolve,										write,		allow
+p,	role:unknown,			/entityresolution/resolve,							  write,		allow
```

Comment:
Not introduced by your PR, but is giving any unknown role access to `/entityresolution/resolve` actually a problem because any authenticated user could reach out to this endpoint to get the idP's entity attributes for any user? Maybe I'm reading it wrong but it seems like the malicious user would just need user names or client ids in their request? @elizabethhealy

Reply:
ya i think that was a "for now" fix when setting it up, and because consumers will have to implement their own ERS we expect them to set up their own policy
we can change it to org admin or something, it should definitely be something more restricted since keycloak restricts who can query clients and users, so maybe a new role that specific to that functionality

Reply:
We can track this within a separate issue. https://github.com/opentdf/platform/issues/1046